processor/src/debug.rs

```diff
@@ -17,12 +17,61 @@ pub struct VmState {
     pub memory: Vec<(u64, Word)>,
 }
 
+impl VmState {
+    pub fn print_stack(&self) {
```

Comment:
TODO: Replace `print`'s with accessors as this is `no_std` code.

Reply:
I would like to avoid putting display logic into `VmState` - I think ti should be handled by consumers of `VmState`. For the purposes of CLI debugger, I think we can keep the display logic in `miden` crate.